apps/frontend/src/components/LoginPage.tsx

```diff
@@ -1,18 +1,18 @@
 import React from "react";
 import {
   Box,
-  Button,
   Flex,
-  FormControl,
-  Input,
-  Stack,
   Heading,
   VStack,
 } from "@chakra-ui/react";
 import { HamburgerIcon, ChevronLeftIcon } from "@chakra-ui/icons";
 
+interface LoginPageProps {
+  children: React.ReactNode;
+}
+
 // Add Header component created by Toco following rebase / merge
-function LoginPage() {
+function LoginPage( { children }: LoginPageProps) {
```

Comment:
Are you passing the `LoginForm` as a prop to `LoginPage`?

Reply:
Thanks for pointing this out.

Before, I was using `children`, but this is a specialized prop that allows me to pass n components. However, in this case, our LoginPage _must_ contain a LoginForm, so it makes more sense to pass in the custom `loginForm` prop.

<img width="683" alt="Screenshot 2024-02-17 at 10 20 01" src="https://github.com/Geek-Collectors-Network/geek-collectors-network/assets/97256979/0f183980-5f73-4371-b992-fb286011d661">

Reply:
Been discussed. :)